utilitiesApp/src/dbLoadRecordsFuncs.cpp

```diff
@@ -203,13 +204,126 @@ epicsShareFunc void dbLoadRecordsList(const char* dbFile, const char* macros, co
 	macDeleteHandle(mh);		
 }
 
+
+/// Create an alias for all records starting with prefix \a recordPrefix by replacing this with \a aliasPrefix
+///
+/// This is a bit like having an alias gateway, but without needing to run a gateway.
+/// It could be extended to only prefix records with e.g. a particular info field present
+///
+/// So if a piece of movable equipoment was being used locally and its PVs/autosave were all set
+/// up in the ME: domain, then aliases for local use could be created using
+/// @code
+///     dbAliasRecords("ME:", "$(MYPVPREFIX)")
+/// @endcode 
+///
+/// @param[in] recordPrefix @copydoc dbAliasRecordsInitArg0
+/// @param[in] aliasPrefix @copydoc dbAliasRecordsInitArg1
+/// @param[in] verbose @copydoc dbAliasRecordsInitArg2
+epicsShareFunc void dbAliasRecords(const char* recordPrefix, const char* aliasPrefix, int verbose)
```

Comment:
I think this should be renamed to something like `dbAliasRecordsPrefix` to make it clearer that it will only work for changing the prefix

Reply:
This function also contains a lot of duplicate code with the other one, really it's just a specific case of the more general function. Could we just add the `(.*)` and `\1` here and call the other function to avoid this duplication? I think the performance cost isn't too bad given this is run once at startup.

Reply:
Happy to change name to `dbAliasRecordsPrefix`. I'd kept the code duplication so the simpler function did not depend on PCRE and hence could be included directly in EPICS base (I had ideas of seeing if there was any interest) or coudl be "borrowed" by somebody without needing PCRE. In some ways the simpler form covers the use case i had originally though about (swap movable equipment -> instrument prefix) and the PCRE one didn't have a use case, but it was easy to add the time as I was doing something else with PCRE